src/sensors/tests/test_irradiancemeter.py

```diff
@@ -0,0 +1,140 @@
+import math
+import random
+
+import numpy as np
+import pytest
+
+import enoki as ek
+import mitsuba
+
+
+def example_shape(radius, x, y, z):
+    from mitsuba.core.xml import load_string
+
+    xml = f"""
+    <shape version='0.1.0' type="sphere">
+        <float name="radius" value="{radius}"/>
+        <transform name="to_world">
+            <translate x="{x}" y="{y}" z="{z}"/>
+        </transform>
+        <sensor type="irradiancemeter">
+            <film type="hdrfilm">
+                <integer name="width" value="1"/>
+                <integer name="height" value="1"/>
+            </film>
+        </sensor>
+    </shape>
+    """
+    return load_string(xml)
+
+
+def test_construct(variant_scalar_rgb):
+    """
+    We construct an irradiance meter attached to a sphere and assert that the
+    following parameters get set correctly:
+    - to_world transform
+    - associated shape
+    - film
+    """
+    radius = 45.2328
+    x = 2
+    y = 5
+    z = 8.3
+    sphere = example_shape(radius, x, y, z)
+    sensor = sphere.sensor()
+
+    assert sensor.shape() == sphere
+    assert ek.allclose(sensor.film().size(), [1, 1])
+
+
+def test_sampling(variant_scalar_rgb):
+    """
+    We construct an irradiance meter attached to a sphere and assert that sampled
+    rays originate at the sphere's surface
+    """
+    from mitsuba.core import Vector3f
+
+    radius = 45.2328
+    x = 2
+    y = 5
+    z = 8.3
```

Comment:
It would be great to parameterize this on a couple position and radii using the pytest `parameterize` feature:
```python
@pytest.mark.parametrize("center", [[0.0, 0.0, 0.0], [-1.0, -1.0, 0.5], [4.0, 1.0, 0.0]])
...
```